README.md

```diff
@@ -1,27 +1,12 @@
 ![tea.xyz](https://tea.xyz/banner.png)
 
-The infuser manages our docker images for platforms we support.
-
-Obviously this cannot include macOS because *Apple*.
-
-The result is a docker image that contains a c compiler and GNU make, with the
-platform’s libc. With this all other tea packages can be built.
-
-Our goal is to provide an image that is just libc and `tea` with a sensible
-(stable) kernel choice but currently we are not there.
-
-# `infuser:latest` (Previously: `infuser:slim`)
+A container with `tea` and its magic installed and functional.
 
 ```sh
-docker buildx build \
-  --pull --push \
-  --tag ghcr.io/teaxyz/infuser:latest \
-  --platform linux/amd64,linux/arm64 \
-  --file infuser/Dockerfile \
-  .
+docker run --rm -it ghcr.io/teaxyz/infuser /bin/bash
```

Comment:
Can remove `/bin/bash` if you set an `ENTRYPOINT` in the `Dockerfile`.

Reply:
oh neat

Reply:
updated

Reply:
This apparently makes it impossible to do anything else with the container, like run a script… unless I'm doing it wrong.

Reply:
Hm. I was pretty sure that if you added a command it overrode `ENTRYPOINT` by default.

Ah, it's `CMD` that is auto-override. https://docs.docker.com/engine/reference/builder/#cmd